src/Widgets/ThumbnailGalleryWidget/ThumbnailGalleryWidgetComponent.js

```diff
@@ -39,7 +39,7 @@ Scrivito.provideComponent("ThumbnailGalleryWidget", ({ widget }) => {
         <div className="row thumbnail-gallery-widget--wrapper">
           {images.map((image, imageIndex) => (
             <Thumbnail
-              key={image.id()}
+              key={`${image.id()}${imageIndex}`}
```

Comment:
* [ ] FYI we construct keys inconsistently. Here we don't have the `-` separator, in line 101 we do.